slideflow/mil/train/_fastai.py

```
@@ -157,7 +159,17 @@ def _build_clam_learner(
     model_size = [n_features] + config_size[1:]
     log.info(f"Training model {config.model_fn.__name__} "
              f"(size={model_size}, loss={config.loss_fn.__name__})")
-    model = config.model_fn(size=model_size, n_classes=batch[-1].shape[-1])
+    
+    if config.net_dim:
+        log.warn("net_dim argument passed is being ignored.")
```

I would just throw an error if the user specifies this and a model that can't use it